dbms/src/Columns/tests/column_vector_perftest.cpp

```diff
@@ -88,6 +100,38 @@ void testScatter(int num_rows, int num_columns, int seconds)
         << std::endl;
 }
 
+template <typename T>
+void testFilter(int num_rows, int num_columns, int seconds)
+{
+    ColumnPtr src = buildColumn<T>(num_rows);
+    auto filter = buildFilter(num_rows, num_columns);
+
+    StopFlag stop_flag = false;
+    std::atomic<Int64> counter = 0;
+
+    auto start = std::chrono::high_resolution_clock::now();
+
+    auto filter_func = [&] {
+        while (!stop_flag.load(std::memory_order_relaxed))
+        {
+            src->filter(filter, -1);
+            counter.fetch_add(1, std::memory_order_relaxed);
+        }
+    };
+
+    std::thread t(filter_func);
```

Comment:
Maybe we just need one thread and test the time costs of handling same filter / data by different method.

Reply:
Have update the perf test and result in pr description, PTAL.